client/src/imageSelector.tsx

```diff
@@ -90,53 +92,42 @@ export default function ImageSelector(props?: ImageSelectorProps) {
     return <>Deploying {selectedImage?.name} to project '{currentContext.project}' on <Link color="inherit" href="#" onClick={() => openInBrowser(currentContext.clusterUrl!!)}>{currentContext.clusterUrl}</Link></>;
   }
 
-  const dockerTheme = useTheme();
   return (
     <>
       <Box margin="15px 0px 15px 0px" display="flex" flexDirection="row">
         <div style={{ flex: '1 1 auto' }}>
-          <Select
-            noOptionsMessage={() => "No images found"}
+          <Autocomplete
+            size="small"
+            // noOptionsMessage={() => "No images found"}
             placeholder="Select an image to deploy"
-            isClearable
-            isSearchable
+            // isClearable
+            // isSearchable
             options={imageOptions}
             onChange={handleSelection}
-            value={imageOptions.filter(image => image.value === selectedImage?.name)}
-            theme={(theme) => ({
-              ...theme,
-              colors: {
-                ...theme.colors,
-                // primary: 'hotpink' // selected input border color
-                primary25: dockerTheme.palette.primary.light, // option highlight color
-                // primary50: 'hotpink' // unknown
-                // primary75: 'hotpink' //unknown 
-                neutral0: dockerTheme.palette.background.default, // background
-                // neutral5: 'hotpink',
-                // neutral10: 'green',
-                // neutral20: 'yellow', // inactive input field border
-                // neutral30: 'blue',
-                // neutral40: 'hotpink', // mouse hoover over x to clean filter and combo button \/ icon
-                neutral50: dockerTheme.palette.text.secondary, // input field placeholder text
-                // neutral60: 'yellow', // selected combo x and \/ icon colors
-                // neutral70: 'hotpink' // unknown
-                neutral80: dockerTheme.palette.text.secondary, // text color for selectd value in input field
-                // neutral90: 'yellow', // unknown
-                // danger: 'hotpink' // unknown
-              },
-            })}
+            // value={imageOptions.filter(image => image.value === selectedImage?.name)}
```

Comment:
when you hit the reload button, the deploy button becomes disabled but the previously selected image is still displayed in the field. setSelectedImage(...) should update the Autocomplete value.

Reply:
![revmove-value-after-reload](https://user-images.githubusercontent.com/620330/176358947-0099c56c-a3f6-4d63-9890-cde8eea19191.gif)